### ISSUE TITLE
Age of Calamity, "Hyrule Castle Town" freezing on load

### ISSUE THREAD

**Issue by CatClanLeader** [yuzu-emu/yuzu#10973]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

Mainline 1471

### Description of Issue

The level "Hyrule Castle Town", a level near the end of the Story Mode, stops loading at 95%. The game still runs its animations and music, but the loading bar is completely stuck and does not continue even after hours of waiting.

I have changed several graphics and CPU settings, tried closing applications, reinstalling the game more than once, and even trying a different graphics card.

### Expected Behavior

The game should load the level just fine. It will probably be laggy due to the games bad optimization even for its own dedicated hardware (Nintendo Switch), but that's a given.

### Reproduction Steps

Press play on the level and wait for it to load.

### Log File

[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/11924856/yuzu_log.txt)

### System Configuration

CPU: 12th Gen Intel(R) Core(TM) i9-12900H 2.50 GHz
GPU/Driver: NVIDIA GeForce RTX 3070 Ti Laptop GPU GDDR6 @ 8GB (256 bits)
RAM: 16GB (15.7 Usable)
OS: Windows 11 Home

**Comment by Squall-Leonhart** [comment]
Caused by https://github.com/yuzu-emu/yuzu/pull/10583
Duplicate https://github.com/yuzu-emu/yuzu/issues/10695

**Comment by aphirst** [comment]
For me, I get freezing on loading, but not specifically this mission. It can happen on any mission and seems to be totally random - somewhere between 1 in 10 and 1 in 20 loading screens. 

I've observed the issue since I started using Yuzu (around 1450 I think - not sure exactly) and it's still present in my current 1483.

Ryzen 5700X, Win10 (12 of 16 cores assigned to VM on Proxmox), Radeon RX 6700 XT (PCIE passthrough to the VM), 24GB of RAM (64GB physical, dual-channel 3200MT/s), latest Adrenaline drivers etc.

I will attach a log next time the issue occurs, to make sure i'm talking about the same basic issue.
